static-analysis/src/main/java/com/linecorp/lich/static_analysis/LichIssueRegistry.kt

```diff
@@ -5,14 +5,19 @@ import com.android.tools.lint.detector.api.CURRENT_API
 import com.android.tools.lint.detector.api.Issue
 import com.linecorp.lich.static_analysis.detectors.OptionalArgumentRequiredDetector
 import com.google.auto.service.AutoService
+import com.linecorp.lich.static_analysis.detectors.LichFactoryDetector
 
 /**
  * A class to register custom [Issue]s
  */
 @AutoService(IssueRegistry::class)
 class LichIssueRegistry : IssueRegistry() {
     override val issues: List<Issue>
-        get() = listOf(OptionalArgumentRequiredDetector.ISSUE)
+        get() = listOf(
+            OptionalArgumentRequiredDetector.ISSUE,
+            LichFactoryDetector.TYPE_ARGUMENT_ISSUE,
+            LichFactoryDetector.OBJECT_ISSUE
+        )
 
     override val api: Int = CURRENT_API
 
```

Comment:
> override val minApi: Int = MIN_API

Is this IssueRegistry really tested on AGP 3.2?
Since the API document of `minApi` says as below, we don't need to override this property, I think.

> The minimum API version this issue registry works with. Normally the
> same as [api], but if you have tested it with older version and it
> works, you can return that level.

Reply:
Right, I think we can remove it. This was to ensure compatibility back in AGP 3.2 where some breaking changes were introduced in lint.
People using AGP lower than 3.5.3 will get a warning asking them to update after removing this

Reply:
> People using AGP lower than 3.5.3 will get a warning asking them to update after removing this

Okay, that would be fine.